docs/site/DEVELOPING.md

```diff
@@ -286,6 +286,48 @@ LoopBack 4 documentation is hosted inside this monorepo in the
 This allows us to change both implementation and the documentation in a single
 pull request.
 
+### Organization of content
+
+LoopBack 4 is a large codebase organized into dozens of packages. We,
+developers, are often tempted to include a lot of details and wider context when
+writing docs. While this is good for capturing the knowledge in a text form, it
+also makes the documentation difficult to follow by people new to LoopBack.
+Newcomers get quickly overwhelmed by the amount of concepts they need to
+understand and all the different packages they need to know about.
+
+To prevent that issue, we have categorized all monorepo packages into two
+groups:
+
+1. Building-blocks are lower-level packages that are typically not consumed by
```

Comment:
`not consumed` -> `not directly consumed` or `not explicitly consumed`?

Reply:
I wonder if we can find a better name for `building blocks`.  Maybe `foundation-level` packages to be symmetric with `framework-level`?

Reply:
Or `library-level`?

Reply:
> `not consumed` -> `not directly consumed` or `not explicitly consumed`?

English is not my mother tongue, but I believe that my current version is grammatically correct while conveying the same meaning.

```
lower-level packages that are typically not consumed by LoopBack projects directly
```

Reply:
> I wonder if we can find a better name for `building blocks`. Maybe `foundation-level` packages to be symmetric with `framework-level`?

I like "foundation-level" + "framework-level".

In https://github.com/strongloop/loopback-next/pull/5659#discussion_r435239706, we were discussing with @hacksparrow that "framework-level" may not be the best description. However, I think with building blocks renamed to "foundation-level", the term "framework-level" is ok.